pyzabbix/sender.py

```diff
@@ -61,7 +61,8 @@ def __init__(self, host, key, value, clock=None):
         self.host = str(host)
         self.key = str(key)
         self.value = str(value)
-        self.clock = clock if clock else str(int(time.time()))
+        if clock is not None:
```

Comment:
```python
if clock:
    self.clock = clock
```
will do pretty much same, in this particular case, because clock will never be 0.